Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:7
```

Comment:
which version of node you are using ?

Reply:
Up to you baby. You can choose any of the "LTS" versions from https://nodejs.org/en/download/

This application only depends on LTS versions

Reply:
i am getting this error while running app.js or may be i dont know how to run this app.

```
NodeSqlApp/app.js:27
app.use((req, res, next) => {
                         ^^
SyntaxError: Unexpected token =>
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
bhumika.pandya@bhumika-pandya-laptop-linux ~/learn_projs/NodeSqlApp (dockerize_me) $ ^Cde app.js 
bhumika.pandya@bhumika-pandya-laptop-linux ~/learn_projs/NodeSqlApp (dockerize_me) $ node --version
v0.12.18
```

Reply:
Try running "npm install" first. This will install all the dependencies. Then run the app